README.md

```diff
@@ -143,4 +143,58 @@ In case you experience problems concerning the submodule checkout, i.e. the clon
 
 # Documentation
 
-TODO
+### Structure
+
+- VulkanLaunchpad.h/VulkanLaunchpad.cpp: main functionality of the framework.
+- Camera.h/Camera.cpp: implementation of an orbit/arcball camera.
+- external: contains the libraries glfw, glm and glslang as git submodules, as well as tinyobjloader and gli.
+
+### Naming Conventions
+
+Function names are in lowerCamelCase starting with the prefix `vkl`. Structs and classes are in UpperCamelCase starting with the prefix `Vkl`. Member variables are mCamelCase, function parameters are lowercase_with_underscores.
+
+### Functionality
+
+The framework needs to be initinilized by calling `vklInitFramework`. Subsequently, it needs to be destroyed via `vklDestroyFramework`. Required extensions can be queried by `vklGetRequiredInstanceExtensions`. Consistent with the Vulkan API, custom configuration structs should best be zero-initialized, although they do not require to expilictly set their instance's type:
+
+    VklSwapchainConfig swapchain_config = {};
+    
+#### General
+
+The basic graphics pipeline created by the framework can be retrieved using `vklGetBasicPipeline`. To set up a custom graphics pipeline you can use `vkl(Create/Destroy)GraphicsPipeline`.
```

Comment:
This part would work better after the code describing the use in a typical rendering loop (lines 166-174), I feel, as this is the order in which we introduce them in the tutorial (the render loop in Part 1, the pipeline creation in Part 2).

Reply:
True. Moved that part a bit down. May I ask one of you @D4id4los or @TheOwlforge to extend the newly created "Graphics Pipelines" section a bit?!

Reply:
But ATTENTION: The whole documentation has been reworked! 😱

Reply:
OK, I'll look into extending the graphics pipeline section.